test/extensions/homotopy_continuation.jl

```diff
@@ -18,16 +24,18 @@ let
     ps = [k1 => 1.0, k2 => 2.0, k3 => 2.0, k4 => 2.0]
     u0 = [:X1 => 2.0, :X2 => 2.0, :X3 => 2.0, :X2_2X3 => 2.0]
 
-    hc_ss = hc_steady_states(rs, ps; u0=u0, show_progress=false)[1]
-    f = ODEFunction(convert(ODESystem, rs))
-    @test f(hc_ss, last.(ps), 0.0)[1] == 0.0
+    # Computes the single steady state, checks that when given to the ODE rhs, all are evaluated to 0.
+    hc_ss = hc_steady_states(rs, ps; u0=u0, show_progress=false)
+    hc_ss = Pair.(unknowns(rs), hc_ss[1])
```

Comment:
hc_steadt_states guarantees that the order is the same as in species(rs). At some point this should have SII support (but would ideally require additional support in SII/MTK/SciMLBase)